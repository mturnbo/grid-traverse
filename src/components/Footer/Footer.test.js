import React from 'react';
import { mount } from 'enzyme';
import FooterComponent from './index';

const title = 'This Is A Test';
const wrapper = mount(<FooterComponent title={title} />);

test('Footer Component Renders', () => {
  expect(wrapper).toMatchSnapshot();
});

test('Footer Component Title', () => {
  expect(wrapper.prop('title')).toEqual(title);
});
