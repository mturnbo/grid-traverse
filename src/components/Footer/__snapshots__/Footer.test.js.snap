// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Renders 1`] = `
<Footer
  title="This Is A Test"
>
  <footer
    className="app-footer"
  >
    <h2>
      This Is A Test
    </h2>
  </footer>
</Footer>
`;
